examples/README.md

```diff
@@ -2,6 +2,7 @@
 - tests are named `[algo].rs`
 - use both yosupo and aizu to test whenever possible because bugs have existed on one of the sites but not the other
 - when using both sites name the files `[algo]_yosupo.rs` and `[algo]_aizu.rs`
+- when there's no problem to test on, test on [hello world](https://onlinejudge.u-aizu.ac.jp/courses/lesson/2/ITP1/all/ITP1_1_A) and name the files `[algo]_stress.rs`
```

Comment:
just for clarity maybe we could use `[algo]_handmade.rs`? I don't really care, just wondering if you think it's more/less clear

Reply:
sure, i can switch it

Reply:
ya now that i think about it, all tests are stress tests

i named it this just to change it up and have a different name from PTC_c++